src/components/Ticket/Ticket.tsx

```diff
@@ -13,7 +13,7 @@ import { Download } from '@mui/icons-material';
 export const Ticket = ({ data }: { data: ITicket }) => {
```

Comment:
Due to ticket minwidth, horizontal scroll appears in mobile screen. May reduce the min-witdth to rectify this issue.

Reply:
done